.artifakt/entrypoint.sh

```diff
@@ -9,9 +9,8 @@ echo "The following build args are available:"
 env
 echo "------------------------------------------------------------"
 
-
 mkdir -p /data/var/log /data/var/uploads /data/var/cache && \
   ln -s /data/var /var/www/html/var && \
-  chown -R www-data:www-data /var/www/html/ /data/var
+  chown www-data:www-data /var/www/html/ /data/var
```

Comment:
@djalal Should do add the chown for 3 directories you created /data/var/log /data/var/uploads /data/var/cache